.github/labeler.yml

```diff
@@ -1,3 +1,6 @@
+ci:
```

Comment:
Should this be?

```suggestion
CI:
```

Reply:
It doesn't really matter. It still works. But I'll change it just because the actual label is capitalised like that.

Reply:
Okay, good to know! And thanks for the change.